docs/source/plugin/anatomy.rst

```diff
@@ -80,6 +80,17 @@ better to limit who can trigger them. There are decorators for that:
   trigger the rule
 * :func:`sopel.plugin.require_owner`: only the bot's owner can trigger the rule
 
+Sometimes, it's not **who** triggers a command that matter, but the bot's
```

Comment:
Since this is contrasting priv check types not identities, I would say "Sometimes it's not the privileges of the user who triggers a command that matter," or switch the emphasis to "triggers". With the latter or as currently written, s/matter/matters/.

Reply:
Done!